src/components/AuthContainer.tsx

```diff
@@ -1,38 +1,42 @@
-import { Card, CardContent, CardHeader, createStyles, makeStyles, Theme } from '@material-ui/core';
+import { Card, CardContent, CardHeader /*, Theme*/ } from '@mui/material';
 import { FC } from 'react';
+// import useClasses from '../hooks/useClasses';
```

Comment:
@jerryhowe - Is there a good reason to leave in the commented out code in this file? Otherwise, I would prefer to take it out to reduce clutter.